components/common/reconcilehelper/util.go

```diff
@@ -16,6 +16,7 @@ import (
 
 // Deployment reconciles a k8s deployment object.
 func Deployment(ctx context.Context, r client.Client, deployment *appsv1.Deployment, log logr.Logger) error {
+	print("reconcilehelper/util.go")
```

Comment:
this one is not marked as covered

![image](https://github.com/user-attachments/assets/8f9a83c0-ca1f-46fc-a611-e735f3725965)